sia/src/lib.rs

```diff
@@ -1,3 +1,4 @@
+pub mod app_client;
```

Comment:
This feels like something that **should** be its own crate since it's specific to `indexd` and should be versioned separately from the core stuff.

Reply:
Went with separate crate for now because it's just too nice. If we want that create to eventually be the same as the Go SDK, we could rename it to something like `sia-app-sdk` considering that it will contain both indexd API code as well as RHP code.